client/my-sites/site-settings/date-time-format/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import DateTimeFormatOptions from './date-time-format-options';
+import DocumentHead from 'components/data/document-head';
+import HeaderCake from 'components/header-cake';
+import { getSelectedSite } from 'state/ui/selectors';
+
+export const DateTimeFormat = ( { translate, site } ) => {
+	const goBack = () => {
+		page( '/settings/general/' + site.slug );
+	};
```

Comment:
maybe I'll start targeting this as soon as I close up some HTTP layer work…

a simple middleware could handle many of the `page()` uses methinks